.github/workflows/go.yml

```diff
@@ -24,36 +28,92 @@ jobs:
       matrix:
         version: ["1.20"]
         command: ["build", "vet", "lint", "test", "testrace"]
-    env:
-      GOPRIVATE: github.com/ServiceWeaver/weaver
     steps:
-      - name: Check out repository
-        uses: actions/checkout@v3
-
-      - name: Set up Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: ${{ matrix.version }}
-          cache: true
-
-      - name: Register weaver deploy key
-        uses: webfactory/ssh-agent@v0.7.0
-        with:
-          ssh-private-key: ${{ secrets.WEAVER_DEPLOY_KEY }}
-
-      - name: Configure git for private modules
-        run: git config --global url."ssh://git@github.com".insteadOf "https://github.com"
-
-      - name: Cache linter
-        uses: actions/cache@v3
-        with:
-          path: ~/go/bin/golangci-lint
-          key: golangci-lint-v1.51.0
-        if: ${{ matrix.command == 'lint' }}
-
-      - name: Install linter
-        run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.0
-        if: ${{ matrix.command == 'lint' }}
-
-      - name: ${{ matrix.command }}
-        run: ./dev/build_and_test ${{ matrix.command }}
+    - name: Check out repository
+      uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ${{ matrix.version }}
+        cache: true
+
+    - name: Cache linter
+      uses: actions/cache@v3
+      with:
+        path: ~/go/bin/golangci-lint
+        key: golangci-lint-v1.51.2
+      if: ${{ matrix.command == 'lint' }}
+
+    - name: Install linter
+      run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.2
+      if: ${{ matrix.command == 'lint' }}
+
+    - name: Build the weaver binary
```

Comment:
Do things still work if we remove this step?

Reply:
It looks like it works!